### DISCUSSION TITLE
Protect my backend access (/admin)

### DISCUSSION THREAD

**Discussion by devPhpMostafa** [luyadev/luya#2154] [Questions & Problems] [answered]
Hello, I need to know how to protect my backend path access as (/admin) alias is easy to be predictable

**Comment by nadar** [comment, marked as answer]
Its not possible to change the `admin` path, but i would recommend to enable secure login instead! Protected the login process with 2fa by email or OTP. See https://luya.io/guide/app/security.html for more options to make protection better
